data_steward/tools/generate_combined_dataset.sh

```diff
@@ -137,12 +137,12 @@ bq update --description "${version} combined clean version of ${rdr_dataset} + $
 bq rm -r -d "${combined_staging}_sandbox"
 bq rm -r -d "${combined_staging}"
 
-dbrowser="${combined}_dbrowser"
+release="${combined}_release"
```

Comment:
I think it's better to keep this `combined_release` since it could be mistaken for the released CDR
```suggestion
combined_release="${combined}_release"
```

Reply:
Updated